README.adoc

```diff
@@ -98,6 +98,29 @@ NOTE: The `+"auth":"<username>:<password>"+` field must be base64-encoded.
 | No
 | The verbosity of logging when running the Kaniko build. It can be one of: `panic`, `fatal`, `error`, `warn`, `info`, `debug`, `trace`. Default is `info`.
 
+|===
+
+== Outputs
+
+[cols="2a,1a,3a",options="header"]
+.Output details
+|===
+
+| Output name
+| Data type
+| Description
+
+| `artifactIds`
+| String
+| The output parameter for artifact IDs.
+
+These artifact IDs values can be accessed in subsequent steps using the outputs context. For example, to return:
+
+* All artifact ID values provided by action in JSON format use: `${{ steps.<action_step_name>.outputs.artifactIds }}`.
+
+* A specific artifact ID value use: `${{ fromJSON(steps.<action_step_name>.outputs.artifactIds).<destination URL>}}`.
```

Comment:
Lets use a hard-coded value in this example to make it clearer to the user what is expected here as the field name to use to get a specific artifact id.